packages/terra-action-footer/tests/wdio/theme/action-footer-theme-spec.js

```diff
@@ -0,0 +1,23 @@
+const {
+  viewports,
+  withActionsThemeables,
+  withoutActionsThemeables,
+} = require('../common');
+
+describe('Action footer theme', () => {
+  describe('Empty', () => {
+    beforeEach(() => browser.url('/#/raw/tests/terra-action-footer/action-footer/empty-action-footer'));
+
+    Terra.should.beAccessible({ viewports });
+    Terra.should.matchScreenshot({ viewports });
+    Terra.should.themeEachCustomProperty(withoutActionsThemeables);
```

Comment:
We should be able to drop 
```
Terra.should.beAccessible({ viewports });
Terra.should.matchScreenshot({ viewports });
```
right?

Reply:
We could, I left the tests exactly as they were just to ensure these test changes were passive in what they were targeting

Reply:
I went ahead and removed these from the theme tests in previous commits,  I think Github is previewing an old commit this thread is on.

https://github.com/cerner/terra-core/pull/2021/commits/ce4c52b6f3bff6eb75c0d111dd4495c6c89ba596

https://github.com/cerner/terra-core/pull/2021/commits/49a2f11b89dba323f140a8728c14bc0d868ddc96

Reply:
whoops 😄